.ci/lanl/gitlab-darwin-ci.yml

```diff
@@ -0,0 +1,114 @@
+variables:
+  SCHEDULER_PARAMETERS: "-pgeneral -t 1:00:00 -N 1 --ntasks-per-node=16"
+  GIT_STRATEGY: clone
+  NPROCS: 4
+
+stages:
+  - build
+  - test
+
+build:intel:
+  stage: build
+  tags: [darwin-slurm-shared]
+  script:
+    - module load intel
+    - git submodule update --init
+    - ./autogen.pl
+    - ./configure CC=icc FC=ifort CXX=icpc --prefix=$PWD/install_test --with-libevent=internal
+    - make -j 8 install
+    - make check
+    - export PATH=$PWD/install_test/bin:$PATH
+    - cd examples 
+    - make
+  variables:
+    SCHEDULER_PARAMETERS: "-pgeneral -t 1:00:00 -N 1 --ntasks-per-node=16"
```

Comment:
This `SCHEDULER_PARAMETERS` seems redudandant (should be covered by top-level `variables:`.

Reply:
yes I'll delete this.  had kept it here to remind me how to do variable sections.